### DISCUSSION TITLE
明天有没有人玩黑猴？

### DISCUSSION THREAD

**Discussion by cyq20091029** [share121/inter-knot#3094] [其他]
<img width="218" alt="截屏2024-08-19 17 19 34" src="https://github.com/user-attachments/assets/dbf52255-bd75-47ef-bee5-27d66c3af58f">

玩的人吱一声

**Comment by Pheathon** [comment]
吱……（无感情）

**Comment by yzysrewrwqr** [comment]
吱~<img src="https://img-static.mihoyo.com/communityweb/upload/1a25d24ab81aaa88543ffad7cbf3f944.png" alt="绝区零_暗中观察" width="75" >

**Comment by LilangAlan** [comment]
吱<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by WhereisFff** [comment]
吱（无感情）

**Comment by Lyou777** [comment]
吱一声

**Comment by MLx-x** [comment]
吱一声

**Comment by wilkerCoder** [comment]
吱<img src="https://bbs-static.miyoushe.com/static/2023/12/28/76bf2420fdd7e884742dc22082828047_1250985772619939913.png" alt="夏沃蕾-补充能量" width="75" >

**Comment by RaphaelHider** [comment]
喵，我要把吱的全吃掉

**Comment by CJHXAMSTLP** [comment]
吱

**Comment by jiaoaodedihuazhixiu** [comment]
吱

**Comment by OttoApocalypse01** [comment]
喵
